cdn-uploader.js

```diff
@@ -0,0 +1,45 @@
+const ftp = require("basic-ftp")
+
+const config = {
+host: process.env.AKAMAI_CDN_HOSTNAME,
+user: process.env.AKAMAI_CDN_USERNAME,
+password: process.env.AKAMAI_CDN_PASSWORD,
+cpCode: process.env.AKAMAI_CDN_CPCODE,
+port: 21,
+basePath: "./builds"
+}
+
+function UploadFilesFromPath(sources) {
+	process.chdir(config.basePath);
```

Comment:
What do you think about receiving basePath as an argument and avoid global state?